design-system.md

```diff
@@ -10,10 +10,9 @@
           <p>HMRC is retiring Assets Frontend following the launch of <a href="https://design-resources-for-hmrc.herokuapp.com/design-library/">HMRC Design Patterns</a>. Assets
              Frontend is still available for services that use it but it will not be developed further. HMRC will
              maintain it and roll out major bug fixes and security patches.</p>
-
           <p>For new projects that use the latest <a href="https://govuk-prototype-kit.herokuapp.com/docs">GOV.UK
              Prototype Kit</a>, use <a href="https://design-resources-for-hmrc.herokuapp.com/design-library/">HMRC Design Patterns</a> to get patterns and code designed
-             specifically for HMRC and for patterns not yet in the <a href="https://design-system.service.gov.uk/">GOV.UK
+             specifically for HMRC and for patterns not yet in the <a href="https://design-system.service.gov.uk/design-system.md">GOV.UK
```

Comment:
I got a 404 when trying to navigate to this. Did you mean the Github repo?

Reply:
Fixed in latest version.